core-api/src/main/java/com/sequenceiq/cloudbreak/api/endpoint/v4/stacks/base/parameter/network/AzureNetworkV4Parameters.java

```diff
@@ -81,6 +92,7 @@ public Map<String, Object> asMap() {
         putIfValueNotNull(map, "networkId", networkId);
         putIfValueNotNull(map, "subnetId", subnetId);
         putIfValueNotNull(map, "databasePrivateDsZoneId", databasePrivateDnsZoneId);
```

Comment:
`Ds` -> `Dns`. Actually would be better replaced with `AzureUtils.DATABASE_PRIVATE_DS_ZONE_ID`.

Please be sure to make the correction in `parse()` as well below.